doc/boot-requirements.md

```diff
@@ -270,8 +270,11 @@
 	- when proposing an new EFI partition
 		- **requires /boot/efi to be a non-encrypted vfat partition**
 		- **requires /boot/efi to be close enough to the beginning of disk**
-		- when aiming for the recommended size
-			- **requires /boot/efi to be exactly 500 MiB large (enough for several operating systems)**
-		- when aiming for the minimal size
-			- **requires it to be at least 256 MiB (min size for FAT32 in drives with 4-KiB-per-sector)**
-			- **requires it to be at most 500 MiB (enough space for several operating systems)**
+		- if aarch64
```

Comment:
How do you generate this changes? I don't expect this output according to the unit tests. In fact, x86 section should not change, but only Raspberry Pi section.

This document can be updated with `rake doc:bootspecs`. Did you do that?

Reply:
Argh. No, of course not; this thing is checked in. So why do we even check that in if this is auto-generated?